README.md

```diff
@@ -1,24 +1,162 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# 📖 Recipes <a name="about-project"></a>
 
-* Database initialization
+The Recipe keeps track of all your recipes and foods. You can save ingredients, view and share your recipes with other users and also generates a shopping list'
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Language and FrameWork</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+<summary>Database Used</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Ruby On Rails**
+- **CanCan**
+- **Devise**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy of this project and run it in your computer, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Ruby
+- Nodejs
+
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+Clone this repository in the desired folder:
+```
+cd my-folder
+git clone https://github.com/solobarine/recipes.git
+```
+
+### Install
+
+To install this project:
+```
+cd recipes
+code .
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+rails server
+```
+
+### Run Tests
+
+To setup rspec, run the following:
+
+```
+rails generate rspec:install
+```
+
+
+To run the project tests, execute the following command:
+
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Solomon Barine Akpuru**
+
+- GitHub: [@solobarine](https://github.com/solobarine)
+- LinkedIn: [solomon-akpuru](https://www.linkedin.com/in/solomon-akpuru)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Deployment.**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
```

Comment:
- [ ] [OPTIONAL] I noticed that the **acknowledgments header section** is missing. I was supposed to make it invalid but since today is Saturday, you don't have a lot of time for review and your readme is quite descriptive and professional. I marked it optional but I would like strongly recommend you add that section to make your readme complete and more professional 😃